pkg/operator/ceph/cluster/osd/spec.go

```diff
@@ -325,6 +325,24 @@ func deploymentName(osdID int) string {
 	return fmt.Sprintf(osdAppNameFmt, osdID)
 }
 
+func (c *Cluster) updateCephConfigVolume(volumes []v1.Volume, nodeName string) []v1.Volume {
```

Comment:
As OSDs are portable, won't per-node setting be a risk?

Reply:
But actually, they could also use it for portable OSDs by creating the configmap named as such:
```
rook-config-override-<pvc-name>
```
Though I don't know a scenario where that would be useful.